dissect/target/plugins/os/windows/wer.py

```diff
@@ -0,0 +1,163 @@
+import re
+from typing import Iterator
+
+from defusedxml import ElementTree
+from dissect.util.ts import wintimestamp
+
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.fsutil import Path
+from dissect.target.helpers.record import DynamicDescriptor, TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+from dissect.target.target import Target
+
+camel_case_pattern = re.compile(r"(\S)([A-Z][a-z]+)")
+camel_case_pattern_2 = re.compile(r"([a-z0-9])([A-Z])")
+camel_case_pattern_3 = re.compile(r"(\w)[.\s](\w)")
+
+
+def _collect_wer_data(wer_file: Path) -> tuple[list[tuple[str, str]], dict[str, str]]:
+    """Parse data from a .wer file."""
+    record_values = {}
+    record_fields = []
+    key = None
+    for line in wer_file.read_text("utf-16").splitlines():
```

Comment:
```suggestion
    for line in wer_file.read_text("utf-16-be").splitlines():
```
Maybe explicitly make it `-be` to make it clear.

Reply:
Talked about this one-on-one: files use a BOM, so we leave this as is.